.githooks/pre-commit

```diff
@@ -101,4 +101,18 @@ then
   done
 fi
 
+# Check Apache License header for source files.
+if [ $NUM_JAVA_FILES_CHANGED -gt 0 ]
+then
+  echo_status "Checking Apache License Header ..."
+  go get -u github.com/google/addlicense
+  addlicense -c "Google LLC" -l apache -check .
```

Comment:
I think this can be a Java-only check without pattern matching - please take a look at the [find](https://ss64.com/bash/find.html) command.

Reply:
done, thanks for the information!